### PR TITLE
Feature/upgrade python3.8

### PR DESCRIPTION
**Pull request by timofurrer**

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -1,11 +1,23 @@
-import os
+"""
+docat
+~~~~~
+
+Host your docs. Simple. Versioned. Fancy.
+
+:copyright: (c) 2019 by docat, https://github.com/randombenj/docat
+:license: MIT, see LICENSE for more details.
```
Comment (fliiiix) [inline backend/app.py]: nice try I will change that shit to BSD

File: backend/tox.ini [hunk 2]
```diff
@@ -9,4 +9,3 @@ deps = flake8
        isort
 commands = flake8 --show-source app.py
            black --check app.py
-           isort --check-only app.py
```
Comment (fliiiix) [inline backend/tox.ini]: why not?

**Comment by fliiiix** [comment]
also thanks :+1:

File: backend/tox.ini (hunk 2 above)
Comment (timofurrer) [reply backend/tox.ini]: It fails, but running `isort` on the fail, doesn't change anything - so I assume misconfiguration .. and I didn't wanna bother ;)

Comment (fliiiix) [reply backend/tox.ini]: weird

**Pull request merged**